lib/language.rb

```diff
@@ -0,0 +1,23 @@
+require 'json'
+
+class Language
+
+    language_file = File.read('./lib/language.json')
+    DATA = JSON.parse(language_file)
```

Comment:
i see this `File.read()` as a dependency for the Language class. could you have it injected in its constructor instead? or make the Language class independent of any file or json provider"

Reply:
Thanks for the feedback @rrugamba, I have injected it through the constructor.